### PR TITLE
Fixing alignment;

### LINKED ISSUE #546: Inline alignment of form elements is off
During updates for the `btn__cell` in the #543, we found the alignment of the inline elements is broken.

https://github.com/eBay/skin/pull/543#issuecomment-459900062

image 21

### PR DESCRIPTION
**Pull request by seangates**
## Description
- fixes height issue for the menu cell
- removes `display: inline-block` from dropdown-related modules (e.g. listbox, menu)
- adjusts some button padding for better alignment
- removes a fixed width since combobox fluidity changed

## Context
Found while attempting a fix for the marketplace issue (#543 )

## References
Fixes #546 

### Before
image
image

### After
image
image

### CODE REVIEW

**Comment by ianmcburnie** [comment]
Seeing some height differences on Firefox:

screen shot 2019-02-06 at 1 51 35 pm

**Comment by ianmcburnie** [comment]
@seangates I guess that was one of the benefits of fixed-height - it made lining these things up a lot easier. Obviously we can't and wont go back to that as our default, but could we just apply the fixed height modifier here? We could stipulate that it's necessary when lining up controls on the same line.

**Comment by seangates** [comment]
Let me try that for the test pages. Then I would just need to update the docs.

**Comment by ianmcburnie** [comment]
If you do that and it works, do you think it then makes sense to revert some (not all) of the changes you've made?

**Comment by seangates** [comment]
Correct. There are some changes still needed.

**Comment by seangates** [comment]
@ianmcburnie I've got it as close as I can get. There's one, teeny, tiny issue with the text on fake button in the large size not being aligned with all the other controls, and I'm not sure how to fix just it, without writing a whole bunch more selectors to target that alignment. Seems hacky for just one pretty minor thing, and cognitively it's hard to tell there's anything wrong. But, please check me on that assumption.

image

**Comment by ianmcburnie** [comment]
Will take a look!

**Comment by ianmcburnie** [comment]
Still seeing issues on Firefox:

screen shot 2019-02-08 at 11 37 01 am

screen shot 2019-02-08 at 11 38 37 am

File: src/less/dropdown/base/dropdown.less [hunk 1]
```diff
@@ -12,7 +12,8 @@ span.menu,
 span.fake-menu,
 span.listbox,
 span.combobox {
-    display: inline-block;
+    display: inline-flex;
```
Comment (ianmcburnie) [inline src/less/dropdown/base/dropdown.less]: I'm curious at to what changing it to flex layout gains us? I didn't think the immediate child elements of those needed to be flex items?

Comment (seangates) [reply src/less/dropdown/base/dropdown.less]: Let me see about reverting this change since I added the `fixed-height` modifiers.

**Comment by seangates** [comment]
Sure, I'll create it.

**Pull request merged**